docs/templates.rst

```diff
@@ -430,6 +430,36 @@ This gives back the results of the parent block::
     {% endblock %}
 
 
+Nesting template extends.
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case of multiple levels of %extends, the childmost {% block %} substitutes into the parent template. For instance,
+
+::
+
+    #a1.tmpl
+    From A1: {% block body %} Want-this-to-be-included. {% endblock %}
+
+    #a2.tmpl
+    {% extends "a1.tmpl" %}
+    {% block body %} Want-this-to-be-skipped. {{super()}} {% endblock %}
+
+    #a3.tmpl
+    {% extends "a2.tmpl" %}
+    {% block body %} Some text {% endblock %}
+
+    #a4.tmpl
+    {% extends "a2.tmpl" %}
+    {% block body %} From A4: {{super.super()}} {% endblock %}
```

Comment:
This example is a great start. I wonder if it can be made any more minimal in such a way that it increases clarity.